sales_team_security/models/res_partner.py

```diff
@@ -11,7 +11,8 @@ class ResPartner(models.Model):
     @api.model
     def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
                         submenu=False):
-        """Patch view to inject the default value for the team_id and user_id.
+        """
+        Patch view to inject the default value for the team_id and user_id.
```

Comment:
Why changing this? docstring can start with the triple quote and the first doc line in the same line